table-api/src/main/java/io/deephaven/api/agg/AggregationDescriptions.java

```diff
@@ -65,6 +65,11 @@ public void visit(Count count) {
         out.put(count.column().name(), "count");
     }
 
+    @Override
+    public void visit(CountWhere countWhere) {
+        out.put(countWhere.column().name(), "countWhere");
```

Comment:
not complete, I would expect to see the where clauses; waht does the formula agg do?